calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| `max_initial_calibration_translation_error` | `double`      | `1.0`                                                   | Maximum allowable translation error in meters in the calibration process, if it is more than the value, a WARNING will show.                      |
+| `max_initial_calibration_rotation_error`    | `double`      | `45.0`                                                  | Maximum allowable rotation error in degree in the calibration process, if it is more than the value, a WARNING will show.                         |
+| `max_number_of_combination_samples`         | `int`         | `10000`                                                 | Maximum number of samples from combinations that are used for cross-validation during the calibration process.                                    |
+
+## Requirements
+
+### Radar reflector
+
+The type of reflector shown in the image below is crucial for our calibration because it has a highly predictable and consistent response to radar. The triangular shape, often composed of three metal plates arranged in a prism form, ensures that the reflector returns signals in specific, predictable ways.
+
+It is recommended that the user build the radar reflector on a tripod, securing it with tape to ensure stability. Additionally, nothing should be attached above the radar reflector; it must be the highest object on the entire calibration target. Additionally, make sure the height of the radar reflector is not larger than the `reflector_max_height` parameter.
+
+<p align="center">
+    <img src="../docs/images/marker_radar_lidar_calibrator/radar_reflector.png" alt="radar_reflector" width="150">
+<p align="center">
+
+## Known issues/limitations
+
+- While extracting the background model, ensure that the radar reflectors are not in the calibration area and that no one is moving around the calibration area.
+
+- While performing the calibration, the calibrator provides a button to delete any mismatched pairs (e.g., an object detected by both radar and lidar). However, some outliers may not be easily detectable by human vision, leading to imperfect results as the calibration proceeds even with these anomalies present. Future enhancements will aim to improve outlier detection, thereby refining the calibration accuracy.
+
+- We have successfully calibrated various sensors with this calibrator, including the Velodyne VLS-128 lidar sensor, Pandar-40P lidar sensor, and ARS408 radar sensor, achieving good calibration results.
+
+## Pro tips/recommendations
+
+- While performing the calibration, it is required that all the reflectors are at the same height with respect to the ground and that both radar and lidar sensors are parallel to the ground.
```

Comment:
There is no mention regarding the corner reflector orientation (I think)

Reply:
All reflectors at the same height, and a ta height matching the radar

Reply:
Do you mean like this https://github.com/tier4/CalibrationTools/pull/164/commits/3f245682304bfc4bcd6c5c87dcc4bed389d9b3e6 ?

Reply:
Actually no.
The requirements itself is: all corner reflectors + the radar itself lie on the same plane. The lidar does not have the same limitation

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/7b7f6fc75d46ad507709e62e9ec03c6ced5782ce